--- src/chcanv.cpp	2014-11-15 16:59:09.854042232 -0600
+++ src/chcanv.cpp	2014-11-15 16:59:09.854042232 -0600
@@ -4796,9 +4796,13 @@
     }
 
     if( b_do_zoom ) {
-        SetVPScale( GetCanvasScaleFactor() / proposed_scale_onscreen );
+        float tmp = GetCanvasScaleFactor() / proposed_scale_onscreen;
+        if(tmp > 0.012)
+        {
+            SetVPScale( tmp );
         Refresh( false );
     }
+    }
 
     m_bzooming = false;

